cfgov/unprocessed/js/modules/util/dom-manipulators.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var domTraverse = require( './dom-traverse' );
+
+/**
+ * Shortcut for creating new dom elements
+ * @param   {string} tag     The html elem to create
+ * @param   {object} options The options for building the elem
+ * @returns {HTMLNode}       The created elem
+ */
+function create( tag, options ) {
+  var elem = document.createElement( tag );
```

Comment:
I'm saying that you are requiring `dom-traverse` when you are only using `queryOne`. Why not breakout the functions in `dom-traverse` into separate modules and then include those modules in `dom-traverse`. This way if you need multiple modules you can include `dom-traverse`, and if you need an individual function you can include just that module.